x/concentrated-liquidity/swaps_test.go

```diff
@@ -355,9 +397,14 @@ func (s *KeeperTestSuite) TestCalcAndSwapOutAmtGivenIn() {
 			// expectedTokenOut: 5000000000.000 + 4321278283.839758464593299720838190090442803542 = 9321278283.8397584645932997208 round down = 9321.278283 usdc
 			expectedTokenIn:  sdk.NewCoin("eth", sdk.NewInt(2000000)),
 			expectedTokenOut: sdk.NewCoin("usdc", sdk.NewInt(9321278283)),
-			expectedTick:     sdk.NewInt(301291),
-			newLowerPrice:    sdk.NewDec(4000),
-			newUpperPrice:    sdk.NewDec(4999),
+			// crosses ticks with fees once, thus DefaultFeeAccumCoins * 3 - DefaultFeeAccumCoins * 2 = DefaultFeeAccumCoins
+			expectedTick:                     sdk.NewInt(301291),
+			expectedLowerTickFeeGrowth:       DefaultFeeAccumCoins.MulDec(sdk.NewDec(2)),
+			expectedUpperTickFeeGrowth:       DefaultFeeAccumCoins.MulDec(sdk.NewDec(2)),
```

Comment:
Same here, not sure how the comment translates to the expected upper and lower tick fee growth

Reply:
ditto for above ^ changed as well!